src/librbd/api/Config.cc

```diff
@@ -80,7 +80,7 @@ struct Options : Parent {
     CephContext *cct = (CephContext *)io_ctx.cct();
 
     for (auto &it : *this) {
-      int r = cct->_conf.get_val(it.first.c_str(), &it.second.first);
+      int r = cct->_conf.get_val(std::string(it.first), &it.second.first);
```

Comment:
Perhaps change ```get_val``` to take a ```string_view``` so this ugly conversion isn't needed (same comment below)?

Reply:
i broke the change into smaller steps. hopefully it'd be easier for reviewers.